docs_src/microservices/application/AppServiceConfigurable.md

```diff
@@ -53,66 +30,105 @@ the configuration in the `configuration.toml` file under the
+
+
+> Note: By default, the configuration provided is set to use MessageBus as a trigger from CoreData. This means you must have EdgeX Running with devices sending data in order to trigger the pipeline. You can also change the trigger to be HTTP. For more on triggers, view the `Triggers`documentation located in the [Application Functions SDK](./ApplicationFunctionsSDK.md#triggers) section.
+
+That's it! Now we can run/deploy this service and the functions pipeline will process the data with functions we've defined.
+
+## Environment Variable Overrides For Docker
+
+Edgex service no longer has docker specific profiles. They now rely on environment variable overrides in the docker compose files for the docker specific differences. The following environment settings are required in the compose files when using App Service Configurable.
+
+```
+EDGEX_PROFILE : [target profile]
+SERVICE_HOST : [service name]
+SERVICE_PORT : [service port]
+MESSAGEBUS_SUBSCRIBEHOST_HOST: edgex-core-data
+CLIENTS_COREDATA_HOST: edgex-core-data
+```
+
+The following is an example docker compose entry for **App Service Configurable**:
+
+```yaml
+  app-service-configurable-rules:
+    image: edgexfoundry/docker-app-service-configurable:1.1.0
+    environment:
+      EDGEX_PROFILE: rules-engine
+      SERVICE_HOST: edgex-app-service-configurable-rules
+      SERVICE_PORT: 48096
+      MESSAGEBUS_SUBSCRIBEHOST_HOST: edgex-core-data
+      CLIENTS_COREDATA_HOST: edgex-core-data
+    ports:
+      - "48096:48096"
+    container_name: edgex-app-service-configurable-rules
+    hostname: edgex-app-service-configurable-rules
+    networks:
+      edgex-network:
+        aliases:
+          - edgex-app-service-configurable-rules
+    depends_on:
+      - data
+```
+
+> *Note: **App Service Configurable** is designed to be run multiple times each with different profiles. This is why in the above example the name `edgex-app-service-configurable-rules` is used for the instance running the `rules-engine` profile.*
+
+## Deploying Multiple Instances using profiles
+
+App Service Configurable was designed to be deployed as multiple instances with different purposes. Since the function pipeline is specified in the `configuration.toml` file, we can use this as a way to run each instance with a different function pipeline. App Service Configurable does not have the standard default configuration at `/res/configuration.toml`. This default configuration has been moved to the `sample` profile. This forces you to specify the profile for the configuration you would like to run. The profile is specified using the `-p/-profile=[profilename]` command line option or the `EDGEX_PROFILE=[profilename]` environment variable override. The profile name selected is used in the service key (`AppService-[profile name]`) to make each instance unique, e.g. `AppService-sample` when specifying `sample` as the profile.
```

Comment:
is it --profile? or -profile?

Reply:
do we want to mention about running the same profile twice? EDIT: I see we have a note on that below, maybe can add a reference to that section since it seems relevant to this section as well.

Reply:
It is eaither --profile or -profile , but we normally use `--` in the docs. Will fix.

Reply:
I moved the note up to here.